middle_end/flambda2/simplify/simplify_expr.ml

```diff
@@ -33,11 +33,12 @@ let rec simplify_expr dacc expr ~down_to_up =
     Simplify_apply_cont_expr.simplify_apply_cont dacc apply_cont ~down_to_up
   | Switch switch ->
     Simplify_switch_expr.simplify_switch ~simplify_let dacc switch ~down_to_up
-  | Invalid _ ->
+  | Invalid { message } ->
     (* CR mshinwell: Make sure that a program can be simplified to just
        [Invalid]. [To_cmm] should translate any [Invalid] that it sees as if it
        were [Halt_and_catch_fire]. *)
```

Comment:
Is this comment still applicable (particularly the part about `Halt_and_catch_fire`) ?

Reply:
I've removed the second sentence.